governance/votes/Executive vote - June 4, 2021.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Template - [Executive Vote] Core Unit June Budget Distributions, Other Changes - June 4, 2021
+
+---
+# [Executive Proposal] Core Unit June Budget Distributions, Other Changes - June 4, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+**Core Unit Budget Distributions - June**
+- **40,000 DAI** will be distributed to **Real World Finance (RWF-001)**.
+- **100,500 DAI** will be distributed to **Risk (RISK-001)**. 
+- **80,000 DAI** will be distributed to **GovAlpha (GOV-001)**.
+- **510,000 DAI** will be distributed to **Protocol Engineering (PE-001)**.
+- **126,117 DAI** will be distributed to **Growth (GRO-001)**.
+- **44,375 DAI** will be distributed to **Content Production (MKT-001)**.
+- **642,135 DAI** will be distributed to **Sustainable Ecosystem Scaling (SES-001)**.
+
+**Liquidations 2.0**
```

Comment:
Don't think it's necessarily worth changing, but did notice the vaults are listed in different order here and in the detail section. All there in both sections :+1: